pkg/codegen/testing/test/testdata/azure-native-nested-types/nodejs/utilities.ts

```diff
@@ -52,3 +52,28 @@ export function getVersion(): string {
 export function resourceOptsDefaults(): any {
     return { version: getVersion() };
 }
+
+/** @internal */
+export function lazy_load(exports: any, module_name: string) {
+	Object.defineProperty(exports, module_name, {
+		enumerable: true,
+		get: function() {
+			return require(`./${module_name}`);
+		},
+	});
+}
+
+/** @internal */
+export function lazy_load_property(exports: any, module_name: string, property: string) {
```

Comment:
Nit: Could you swap the args `module_name` and `property` here?
Since we know `Object.defineProperty(x, y)` defines `x.y`, I read the function arguments to `lazy_load_property` the same way. If you swap `module_name` and `property`, a reader can interpreter `lazy_load_property` as `exports.property = require(module_name)`

```suggestion
export function lazy_load_property(exports: any, property: string, module_name: string) {
```

Reply:
Done.